modules/custom/d_geysir/d_geysir.module

```diff
@@ -61,6 +61,13 @@ function d_geysir_field_widget_link_attributes_form_alter(&$element, &$form_stat
   $element["options"]["attributes"]["#open"] = 0;
 }
 
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function d_geysir_field_widget_form_alter(&$element, &$form_state, $context) {
```

Comment:
Mała prośba żeby dopisać komentarz po co to wszystko :-)

Reply:
done :)